uint8 grasping_force # in percentage (0..100)
---
bool success
string message
